test/lib/user.test.js

```diff
@@ -219,48 +219,4 @@ describe('User Model', () => {
             assert.calledOnce(tokenFactoryMock.list);
         });
     });
-
```

Comment:
We don't have a validateToken on the tokens any more, in the PR after this (ready for once this one is finished review) I added a way to get a user by token value, so the login flow for that will be the same as getting a user with oauth and then giving them a 12h token.